packages/dds/tree/src/simple-tree/api/customTree.ts

```diff
@@ -69,21 +69,21 @@ export enum KeyEncodingOptions {
 	 * Use property keys.
 	 * @remarks
 	 * Supported for import and export.
-	 * {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields|Unknown optional fields} will be omitted when using property keys.
+	 * {@link ObjectSchemaOptions.allowUnknownOptionalFields|Unknown optional fields} will be omitted when using property keys.
 	 */
 	usePropertyKeys = "usePropertyKeys",
 	/**
-	 * Use stored keys, and include {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields|Unknown optional fields}.
+	 * Use stored keys, and include {@link ObjectSchemaOptions.allowUnknownOptionalFields|Unknown optional fields}.
 	 * @remarks
 	 * Currently only supported for export.
 	 */
 	allStoredKeys = "allStoredKeys",
 	/**
-	 * Use stored keys but do not include {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields|Unknown optional fields}.
+	 * Use stored keys but do not include {@link ObjectSchemaOptions.allowUnknownOptionalFields|Unknown optional fields}.
```

Comment:
Nit (existing, but might as well clean up while we're here, if you don't mind):
```suggestion
	 * Use stored keys but do not include {@link ObjectSchemaOptions.allowUnknownOptionalFields | unknown optional fields}.
```